code/src/App.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react'
 import data from './data.json'
-
-console.log(data)
+import playlist_data from './stretch-goal.json'
+import Header from './components/Header'
+import Album from './components/Album'
+import Playlists from './components/Playlists'
 
 export const App = () => {
+  const AlbumItems = data.albums.items
+  const PlaylistItems = playlist_data.playlists.items
```

Comment:
JavaScript variables should have camelCase: `const newReleases = data.albums.items`